Yoast/wordpress-seo
# Inclusive language: change string for the inclusive language orange bullet in pre publish box

## Context


* In case of an overall orange bullet for inclusive language, the string in the pre-publish box says “OK” (see screenshot). Unlike for the SEO and readability analysis, an orange bullet for the inclusive language analysis doesn’t necessarily mean it’s “OK”. But rather, that potentially non-inclusive language was detected and improvements may be needed.
* Styling changes were made to make sure that styling was correct if feedback steps were wrapped over multiple lines.

## Summary


This PR can be summarized in the following changelog entry:

* Changes the publish box feedback from 'OK' to 'Potentially non-inclusive' for _Inclusive Language_ analysis.

## Relevant technical choices:

* A css flexbox was added to the image and the feedback text to make the text wrap correctly.
* NOTE: the code for the publish box in classic editor lives in PHP and the code for the publish box in the block editor lives in Java Script.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Install and activate yoastSEO free.
* Make sure that _inclusive language assessment_ is enabled.

##### Scenario 1: Test block-editor
* Open a post/page/... in block editor.
* Add the word 'policemen' (or any other potentially non inclusive word if you feel adventurous.)
* Go to `settings` -> `Post` (see screenshot)
<img width="1356" alt="Screenshot 2022-12-14 at 09 24 10" src="https://user-images.githubusercontent.com/26040749/207549555-44570cca-dc17-4a30-b952-fb94fef9b630.png">

* Make sure that the feedback for inclusive language is an orange bullet and the text `Potentially non-inclusive`
* Make sure that it looks like in the screenshot above. Especially look at the alignment and text wrapping.

* Click on publish once.
* Make sure that te feedback for inclusive language is an orange bullet and the text `Potentially non-inclusive`.
* Make sure that it looks like in the screenshot below. Especially look at the alignment and text wrapping.
 
<img width="1356" alt="image" src="https://user-images.githubusercontent.com/26040749/207551160-4402ed89-5c5d-462f-a898-d1dc1dec5b93.png">

* Add the word `midget` to your post
* Make sure that the publish box feeback for inclusive language gives a red smiley and the following feedback: `Needs improvement`

* Remove all (potentially) non inclusive terms from the post. 
* Make sure that the publish box feedback for inclusive language gives a green smiley and the following feedback: `Good `

##### Scenario 2: Test classic editor
* Open a post/page/... in block editor.
* Add the word 'policemen' (or any other potentially non inclusive word if you feel adventurous.)
* Make sure that in the publish box, you get an orange yoast logo and the following feedback: `Potentially non-inclusive`
* Make sure that it looks like in the screenshot above. Especially look at the alignment and text wrapping.
<img width="1355" alt="image" src="https://user-images.githubusercontent.com/26040749/207554935-41beae38-f90d-4c32-86d1-09ba5c8e86dc.png">

In Elementor we do not have a publish box.

#### Relevant test scenarios


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

* This PR affects the inclusive language part of the publish box in block editor and classic editor.

## UI changes


## Other environments


## Documentation


## Quality assurance


## Innovation


Fixes #https://yoast.atlassian.net/browse/PC-996?atlOrigin=eyJpIjoiZDQ2MTZlNDUwMjBjNGYwYThiZmUwNzNiMTc1YWQxYzciLCJwIjoiaiJ9
